dbt/include/firebolt/macros/materializations/table.sql

```diff
@@ -2,21 +2,19 @@
 {# Note that a nearly identical materialization appears
    in table.sql #}
   {%- set identifier = model['alias'] -%} {# alias comes from where? #}
-  {%- set old_relation = adapter.get_relation(database=database,
-                                              schema=schema,
-                                              identifier=identifier) -%}
   {%- set target_relation = api.Relation.create(database=database,
                                                 schema=schema,
                                                 identifier=identifier,
                                                 type='table') -%}
```

Comment:
Can you write some comments somewhere in this code to explain what it's doing? I'm really not clear, so I'm sure a more naive reader would have even more problems.

Reply:
Happy to flesh out a comment but this code isn't going to live long. As a start, is the description given in this PR helpful?

Reply:
Yeah, that's helpful. I think I missed it when I clicked on the link in the email I received. Maybe when the page loaded it scrolled past the PR comment. Or maybe I was so eager to get to the code that I missed it, in which case, just chalk it up to me being me, I guess. 😞 

Out of curiosity, what's actually causing the problem?

Reply:
check out https://github.com/dbt-labs/dbt-core/issues/4187. it seems there's a weird approximate match failure happening that'll take another week to iron out

Reply:
Just added some context inline with jinja comment string.